araig_interpreters/scripts/goal_interpreter

```diff
@@ -23,7 +23,7 @@ if __name__ == "__main__":
     goal.target_pose.pose.position.z = rospy.get_param(config_goal + '/position/z')
 
     yaw = rospy.get_param(config_goal + '/yaw')
-    rot = Rotation.from_euler('xyz', [0, 0, yaw], degrees=True)
```

Comment:
Well I wanted to keep it in degrees in the yaml because it is easier for us to immediately understand